src/helpers/formControl.test.ts

```diff
@@ -0,0 +1,21 @@
+import { resolveInputIdFromName } from './formControl';
+
+describe('resolveInputIdFromName', () => {
+  it('returns ID if ID attribute is set', () => {
```

Comment:
How about add another level of describe to put the context?
```suggestion
  describe('if ID attribute is set', () => {
    it('returns ID', () => {
      # ...
    }
  })
```

Reply:
I agree with @hanam1ni but for the `describe` block it should be like the pattern with `given ...`.
I think we can apply it to others also 🙌

Reply:
Updated on [a941343](https://github.com/rosle/react-nimble-survey/pull/14/commits/a941343095b52e3d16beb16827a391dd2e212b00) ✨